sotodlib/io/bookbinder.py

```diff
@@ -1034,6 +1090,14 @@ def compile_mfile_dict(self):
              'end_time':   self._end_time.time/core.G3Units.s,
              'n_frames':   self.frame_num,
              'n_samples':  self.sample_num}
+
+        if self.hwp_loader is not None:
+            if self.hwp_loader.hwp_angle_interp is not None:  # If there's data
+                hwp_rate = float(np.mean(self.hwp_loader.data['slow']['hwp_rate']))
```

Comment:
What’s the reason for the `float()` here?

Reply:
I had to add this because mean returns a `np.float`, which isn't formatted well in yaml

Reply:
In that case, could you add a brief comment for future viewers?